packages/useTrie/src/index.ts

```diff
@@ -43,7 +45,7 @@ class Trie implements ITrie {
 
   constructor(
     words: Word[] = [],
-    private isCaseInsensitive: boolean = true,
+    private isCaseInsensitive = true,
```

Comment:
The type is inferred by the default value `true`.

Reply:
Is a type inference a recommended way by chance?
It does look simpler & more readable

Reply:
If it can be inferred, it's not necessary to explicitly type it.

Reply:
I see. Thank you for the clarification 🙂
I being explicit was always the norm. I will update the rest of the code-base, as well.